BridgeAppSDK/ORKResult+SBAExtension.swift

```diff
@@ -280,12 +280,10 @@ class AnswerKeyAndValue: NSObject {
     }
 }
 
-protocol ORKQuestionResultAnswerJSON {
+public protocol ORKQuestionResultAnswerJSON {
     func jsonSerializedAnswer() -> AnswerKeyAndValue?
 }
 
-private var _timeOnlyFormatter: NSDateFormatter? = nil
-
 extension ORKQuestionResult: ORKQuestionResultAnswerJSON {
```

Comment:
Ah, right. ORKQuestionResult and  ORKQuestionResultAnswerJSON are public therefore this is automatically public. confusing.